x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java

```diff
@@ -208,7 +209,7 @@ public void testInvalidateAllTokensForUser() throws Exception {
                 SecuritySettingsSourceField.TEST_PASSWORD.toCharArray()), SECURITY_REQUEST_OPTIONS);
         }
         InvalidateTokenResponse invalidateResponse = restClient.security().invalidateToken(
-            new InvalidateTokenRequest(null, null, null, SecuritySettingsSource.TEST_USER_NAME),
+            InvalidateTokenRequest.userTokens(SecuritySettingsSource.TEST_USER_NAME),
```

Comment:
Not suggesting for you to change it in this PR since it is user visible, but the method name `userTokens` is confusing.

Reply:
I agree - I thought the same thing when I converted all these calls.